### ISSUE TITLE
Default map provider should use https

### ISSUE THREAD

**Issue by scottybo** [GeneaLabs/nova-map-marker-field#14]
I noticed that the default provider for this field is using http instead of https, forcing us to always define the tile provider (even though we're using OSM)

```php
            MapMarker::make("Location")
                ->tileProvider('https://{s}.tile.openstreetmap.org/{z}/{x}/{y}.png'),
```

**Comment by mikebronner** [comment]
@scottybo  Thanks for catching this!

**Comment by mikebronner** [comment]
This should now be fixed in release 0.1.6. Please let me know how it works for you.
